modules/tabs/js/tabs_directive.js

```diff
@@ -76,6 +77,13 @@
 
         $scope.$watch(function()
         {
+            return lxTabs.position;
```

Comment:
Whatcher is for keeping code consistency reason.
You have used watchers for lxTabs.activeTab and lxTabs.links, so, I've decided to implement it in existing style (in accordance with the principle of minimal changes).